src/pages/Plant.js

```diff
@@ -9,18 +9,28 @@ const Plant = (props) => {
           <img src={props.plant.species.img} alt="plantimage"/>
           <h1>{props.plant.name}</h1>
           <section className= "details">
-          <h4>Frequency: {props.plant.species.frequency}</h4>
-          <h4>Sunlight: {props.plant.species.sunlight}</h4>
-          <h4>Temperature: {props.plant.species.temperature.min} - {props.plant.species.temperature.max}</h4>
+            <h4>
+              <i class="far fa-calendar"></i>  Frequency: {props.plant.species.frequency}
+            </h4>
+            <h4>
+              <i class="fas fa-sun"></i>  Sunlight: {props.plant.species.sunlight}
+            </h4>
+            <h4>
+              <i class="fas fa-temperature-high"></i> Temperature: {props.plant.species.temperature.min} - {props.plant.species.temperature.max}
+            </h4>
           </section>
           <hr/>
           <form><h4>Description: {props.plant.species.description}</h4></form>
         </article>
-        <button onClick={() => {
+         <button onClick={() => {
             props.deletePlant(plant)
           }}>
             Delete Plant
-          </button>
+        </button>
+        <button onClick={() => {
+            props.selectedPlant(plant)
```

Comment:
selectedPlant is not a function, nor is it passed as a prop to this component. The selectedPlant is already set to the plant here from the MyGarden component. This line can be removed. This could probably just be addressed with a Link tag around the button instead of through the onClick() function.